clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -355,6 +355,8 @@ def err_drv_expecting_fsycl_with_sycl_opt : Error<
   "'%0' must be used in conjunction with '-fsycl' to enable offloading">;
 def err_drv_fsycl_with_c_type : Error<
   "'%0' must not be used in conjunction with '-fsycl', which expects C++ source">;
+def err_drv_fsycl_with_pch : Error<
+  "Precompiled header generation is not supported with '-fsycl'">;
```

Comment:
For this case, we aren't generating PCH files, we are consuming.

Reply:
My bad. Will fix it.